osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -53,6 +53,8 @@ public OverlayRulesetTabItem(RulesetInfo value)
                         Origin = Anchor.Centre,
                         Anchor = Anchor.Centre,
                         Text = value.Name,
+                        Font = OsuFont.GetFont(size: 18),
+                        ShadowColour = new Color4(0, 0, 0, 0.75f)
```

Comment:
Slightly more readable:
```suggestion
                        ShadowColour = Color4.Black.Opacity(0.75f)
```